src/cf.data.pre

```diff
@@ -5473,7 +5473,9 @@ DOC_START
 
 	This is where general information about Squid behavior goes. You can
 	increase the amount of data logged to this file and how often it is
-	rotated with "debug_options"
+	rotated with "debug_options".
+	Caching the general information would introduce extra I/O and may cause performance issue 
+	when the workload is heavy. The cache_log can be disabled by setting the configuration to cache_log /dev/null 
```

Comment:
This log is not optional. /dev/nulll does *not* disable it, and is a very bad idea.

By default this log receives only critical and important messages admin need to be aware of. Outputs here are controlled by debug_options directive. So performance issues should be mentioned in debug_options documentation.

Reply:
Thanks for your answer. I was search on google for how to diable the cache_log and some answer suggest that I can disable it with /dev/null. I am not very familiar with the detail implementation of cache_log. Why it is a bad idea to set it to /dev/null?

Reply:
> Why it is a bad idea to set cache_log to /dev/null?

... because you will not see potentially critical Squid warnings and errors, including those about misconfigurations and environmental problems (unless you properly setup/enable syslog recording of those messages).

Reply:
Thanks for your answer! It is very useful.